gemfiles/rails_5_0.gemfile

```diff
@@ -6,7 +6,7 @@ gem "rack", "~> 1", platforms: [:ruby_19, :ruby_20, :ruby_21]
 gem "rake"
 gem "appraisal"
 gem "activerecord", "~> 5.0.6"
-
+gem "i18n", "~> 1.4"
```

Comment:
The master suite is currently green as of yesterday so I don't think we need this change.